### PR TITLE
Proof of concept, just to be sure we can intercept astroquery calls

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

File: renkuaqs/plugin.py [hunk 1]
```diff
@@ -14,38 +18,36 @@
...
+from renku.core.models.provenance.provenance_graph import ProvenanceGraph
+from renku.core.errors import RenkuException
 
 from prettytable import PrettyTable
 from deepdiff import DeepDiff
 
-from .config import AQS_DIR
-#from aqsconverters.models import Run
+from aqsconverters.models import Run
+from aqsconverters.io import AQS_DIR, COMMON_DIR
```
Comment (burnout87) [inline renkuaqs/plugin.py:38]: Are these necessary? For me, they keep generating errors

**Review by burnout87 — CHANGES_REQUESTED** [review]

File: renkuaqs/plugin.py (hunk 1 above)
Comment (volodymyrss) [reply renkuaqs/plugin.py:38]: they will be eventually necessary and can be installed from https://github.com/oda-hub/aqsmodel-converters

**Comment by volodymyrss** [comment]
ok, but could you please not push to this branch, review it as basis, and so we proceed with it and future PRs you will also make?

**Comment by burnout87** [comment]
Just a little suggestion for the tests, and to get the aqsmodel-converters repository.

**Comment by volodymyrss** [comment]
> Just a little suggestion for the tests, and to get the aqsmodel-converters repository.

Thanks! but it is a bit difficult to track what is being suggested, and comment on it, if you straight push in the branch

**Review by burnout87 — APPROVED** [review]

**Comment by volodymyrss** [comment]
cool, but could we agreed **not** to do this push on suggested branch, and open another PR?

**Pull request merged**